### ISSUE TITLE
Missing requirements for director

### ISSUE THREAD

**Issue by GrexAut** [jjethwa/icinga2#197]
Hi,

following modules are required for director, that are missing:
reactbundle
ipl
incubator

see:
![grafik](https://user-images.githubusercontent.com/5990312/66045377-e92a7280-e523-11e9-80cb-98c3b195286c.png)

**Comment by jjethwa** [comment]
Thanks for your work on this @GrexAut  I worked on getting the container updated for the updated requirements for icinga2 itself, but did not see that there were director specific ones.  Looks like we'll need to add those module installs (example: https://www.kickstarter.com/projects/1391511229/rollux-the-most-versatile-2-in-1-suitcase-around ) to https://github.com/jjethwa/icinga2/blob/master/content/opt/setup/61-icingaweb2-director

Is this something you'd like to work on and submit a PR for?

**Comment by svenakela** [comment]
They are added in the PR I did a few minutes ago.

**Comment by MuskratAggie** [comment]
I did some Dockerfile hacking and was able to get the new modules installed...but being the n00b that I am, I didn't understand how to get Supervisor to enable them when you start up the containers.  No problem, was able to enable them in the web GUI.  After that, I ran into another issue that Icinga introduced...an Icinga Director Background Daemon service that also has to be running now ([https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/75-Background-Daemon.md](https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/75-Background-Daemon.md)). I also found that I wasn't able to update the DB schema, but I don't think that has anything to do with this project...my DB may be the issue.  As always, thanks for the great work!!
